<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="10" passed="2" failed="6" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-03-30T10:52:35Z" name="CRMPRO Application" finished-at="2022-03-30T10:57:01Z" duration-ms="265861">
    <groups>
    </groups>
    <test started-at="2022-03-30T10:52:35Z" name="Application Page Functionalities" finished-at="2022-03-30T10:57:01Z" duration-ms="265861">
      <class name="com.test.HomePageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.test.HomePageTest@1daf3b44]" started-at="2022-03-30T10:53:10Z" name="setup" finished-at="2022-03-30T10:53:18Z" duration-ms="7847" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateHomePage()[pri:0, instance:com.test.HomePageTest@1daf3b44]" started-at="2022-03-30T10:53:18Z" name="validateHomePage" finished-at="2022-03-30T10:53:44Z" duration-ms="26008" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.pages.LoginPage.login(LoginPage.java:145)
at com.test.HomePageTest.validateHomePage(HomePageTest.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateHomePage -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.test.HomePageTest@1daf3b44]" started-at="2022-03-30T10:53:44Z" name="teardown" finished-at="2022-03-30T10:53:45Z" duration-ms="914" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.test.HomePageTest@1daf3b44]" started-at="2022-03-30T10:54:16Z" name="setup" finished-at="2022-03-30T10:54:24Z" duration-ms="7716" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateClickingOnContactsLink()[pri:1, instance:com.test.HomePageTest@1daf3b44]" started-at="2022-03-30T10:54:24Z" name="validateClickingOnContactsLink" finished-at="2022-03-30T10:55:13Z" duration-ms="49775" status="FAIL">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Must provide a location for a move action.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Must provide a location for a move action.
at org.openqa.selenium.interactions.MoveMouseAction.<init>(MoveMouseAction.java:37)
at org.openqa.selenium.interactions.Actions.moveToElement(Actions.java:362)
at com.pages.HomePage.clickOnContactsLink(HomePage.java:89)
at com.test.HomePageTest.validateClickingOnContactsLink(HomePageTest.java:62)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateClickingOnContactsLink -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.test.HomePageTest@1daf3b44]" started-at="2022-03-30T10:55:14Z" name="teardown" finished-at="2022-03-30T10:55:15Z" duration-ms="833" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.test.HomePageTest -->
      <class name="com.test.ContactsPageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.test.ContactsPageTest@7fd8c559]" started-at="2022-03-30T10:53:45Z" name="setup" finished-at="2022-03-30T10:53:54Z" duration-ms="8654" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateUserOnContactsPage()[pri:0, instance:com.test.ContactsPageTest@7fd8c559]" started-at="2022-03-30T10:53:54Z" name="validateUserOnContactsPage" finished-at="2022-03-30T10:54:05Z" duration-ms="11300" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateUserOnContactsPage -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.test.ContactsPageTest@7fd8c559]" started-at="2022-03-30T10:54:05Z" name="teardown" finished-at="2022-03-30T10:54:06Z" duration-ms="979" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.test.ContactsPageTest@7fd8c559]" started-at="2022-03-30T10:55:17Z" name="setup" finished-at="2022-03-30T10:55:24Z" duration-ms="7679" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateUserIsAbleToCreateANewContact(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.test.ContactsPageTest@7fd8c559]" started-at="2022-03-30T10:55:24Z" name="validateUserIsAbleToCreateANewContact" data-provider="testData" finished-at="2022-03-30T10:55:50Z" duration-ms="25426" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Subhankar]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sahoo]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[TCS]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[abc@256]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.pages.LoginPage.login(LoginPage.java:145)
at com.test.ContactsPageTest.validateUserIsAbleToCreateANewContact(ContactsPageTest.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateUserIsAbleToCreateANewContact -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.test.ContactsPageTest@7fd8c559]" started-at="2022-03-30T10:55:50Z" name="teardown" finished-at="2022-03-30T10:55:51Z" duration-ms="856" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.test.ContactsPageTest@7fd8c559]" started-at="2022-03-30T10:55:51Z" name="setup" finished-at="2022-03-30T10:56:00Z" duration-ms="8427" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateUserIsAbleToCreateANewContact(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:1, instance:com.test.ContactsPageTest@7fd8c559]" started-at="2022-03-30T10:56:00Z" name="validateUserIsAbleToCreateANewContact" data-provider="testData" finished-at="2022-03-30T10:56:24Z" duration-ms="24124" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Amrit]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Srivastava]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Eli Lilly]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[uvc@456]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.pages.LoginPage.login(LoginPage.java:145)
at com.test.ContactsPageTest.validateUserIsAbleToCreateANewContact(ContactsPageTest.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateUserIsAbleToCreateANewContact -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.test.ContactsPageTest@7fd8c559]" started-at="2022-03-30T10:56:24Z" name="teardown" finished-at="2022-03-30T10:56:25Z" duration-ms="876" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.test.ContactsPageTest -->
      <class name="com.test.LoginPageTest">
        <test-method is-config="true" signature="allureDirectoryEmpty()[pri:0, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:52:35Z" name="allureDirectoryEmpty" finished-at="2022-03-30T10:52:35Z" duration-ms="247" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- allureDirectoryEmpty -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:52:35Z" name="setup" finished-at="2022-03-30T10:53:07Z" duration-ms="32467" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateUserInLoginPageOfCRMApplication()[pri:0, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:53:08Z" name="validateUserInLoginPageOfCRMApplication" finished-at="2022-03-30T10:53:09Z" duration-ms="740" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateUserInLoginPageOfCRMApplication -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:53:09Z" name="teardown" finished-at="2022-03-30T10:53:10Z" duration-ms="940" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:54:06Z" name="setup" finished-at="2022-03-30T10:54:14Z" duration-ms="7599" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateLoginPageTitle()[pri:1, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:54:14Z" name="validateLoginPageTitle" finished-at="2022-03-30T10:54:14Z" duration-ms="45" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ CRMPRO - CRM software for customer relationship management, sales, and support.] but found [CRMPRO - CRM software for customer relationship management, sales, and support.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ CRMPRO - CRM software for customer relationship management, sales, and support.] but found [CRMPRO - CRM software for customer relationship management, sales, and support.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.test.LoginPageTest.validateLoginPageTitle(LoginPageTest.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginPageTitle -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:54:15Z" name="teardown" finished-at="2022-03-30T10:54:16Z" duration-ms="791" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:56:25Z" name="setup" finished-at="2022-03-30T10:56:34Z" duration-ms="8427" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateUserLogin()[pri:2, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:56:34Z" name="validateUserLogin" finished-at="2022-03-30T10:56:59Z" duration-ms="25527" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.pages.LoginPage.login(LoginPage.java:145)
at com.test.LoginPageTest.validateUserLogin(LoginPageTest.java:82)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateUserLogin -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.test.LoginPageTest@60859f5a]" started-at="2022-03-30T10:57:00Z" name="teardown" finished-at="2022-03-30T10:57:01Z" duration-ms="874" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.test.LoginPageTest -->
    </test> <!-- Application Page Functionalities -->
  </suite> <!-- CRMPRO Application -->
</testng-results>
